cluster/base/Makefile

```diff
@@ -0,0 +1,12 @@
+base-image:
+		mkdir -p staging/cc
+		cp ../../WORKSPACE staging/
+		cp -r -p ../../cc/ staging/
+		rm -rfd staging/cc/tensorflow
```

Comment:
this bit fixes #242 by preventing any host-compiled .so's from getting copied into the staging directory.